requirements/lite/base.txt

```diff
@@ -1,6 +1,7 @@
 # NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
+numpy>=1.17.2, <1.23.1
```

Comment:
Will PL still need the numpy installation?

Reply:
Yes, there seem to be usages across several Lite unrelated files

Reply:
The pin is for PIL?

Reply:
Same pin as in PL, no changes.